design-system/materials/internals/definition.yaml

```diff
@@ -22,7 +22,7 @@
 # files in different format (css, json, js, ...).
 #
 # The names of decisions in this file need to follow this format:
-#   <attribute>-for-<component-group>-when-<state>-
+#   <attribute>-for-<component-group>-as-<variant>-when-<state>
```

Comment:
Should we include the same examples we have in the generate script?

https://github.com/commercetools/ui-kit/blob/0a42178c2c733cf31120fa208db182399887677e/design-system/scripts/generate-design-tokens.js#L39-L51

Reply:
Yes, was thinking about it as well. I'll add it :)

Reply:
Added [3a4141a](https://github.com/commercetools/ui-kit/pull/2317/commits/3a4141a69b03e085e3adffd61f7456a8fe0832ee) ✅